students/seyong/users/views.py

```diff
@@ -29,10 +29,13 @@ def post(self, request):
             if User.objects.filter(email = email).exists():
                 return JsonResponse({"message": "EMAIL_ALREADY_EXISTS"}, status = 400)
             
+            hashed_password       = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+            decoded_password      = hashed_password.decode('utf-8')
```

Comment:
어차피 아래에서 디코딩 해줄거라면 굳이 두 번에 걸쳐서 처리할 필요가 없어보입니다
```suggestion
            hashed_password       = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Reply:
수정하였습니다 감사합니다!!